netbox/extras/models/search.py

```diff
@@ -57,6 +57,9 @@ class Meta:
         ordering = ('weight', 'object_type', 'value', 'object_id')
         verbose_name = _('cached value')
         verbose_name_plural = _('cached values')
+        indexes = (
+            models.Index(fields=('object_type', 'object_id')),
```

Comment:
We can specify a name for the index to avoid Django attaching random cruft to it.
```suggestion
            models.Index(fields=('object_type', 'object_id'), name='extras_cachedvalue_object'),
```